.gitignore

```diff
@@ -0,0 +1,2 @@
+# IDEs folders
+.idea
```

Comment:
Unrelated.

```suggestion
```

Reply:
For sure. This is done https://github.com/disney/meteor-base/pull/168/commits/1754aa13cf4f20596eb3b63e0a36fdd93bbf6d7b Thanks!